robusta_krr/core/models/config.py

```diff
@@ -68,6 +69,13 @@ def validate_namespaces(cls, v: Union[list[str], Literal["*"]]) -> Union[list[st
         if v == []:
             return "*"
 
+        return [val.lower() for val in v]
+
+    @pd.validator("resources")
+    def validate_resources(cls, v: Union[list[str], Literal["*"]]) -> Union[list[str], Literal["*"]]:
+        if v == []:
+            return "*"
+
```

Comment:
do we validate that all the resources are ones we support? if someone mispells deployment what will happen?